Grammar/python.gram

```diff
@@ -239,7 +246,14 @@ star_named_expression[expr_ty]:
 named_expression[expr_ty]:
     | a=NAME ':=' b=expression { _Py_NamedExpr(CHECK(set_expr_context(p, a, Store)), b, EXTRA) }
     | expression
-annotated_rhs[expr_ty]: yield_expr | expressions
+
+annotated_rhs[expr_ty]: yield_expr | star_expressions
+
+expressions[expr_ty]:
+    | a=expression b=(',' c=expression { c })+ [','] {
+        _Py_Tuple(CHECK(seq_insert_in_front(p, a, b)), Load, EXTRA) }
```

Comment:
Can this not be expressed with the `'.'` operator?

Reply:
I don't think so, since there is a `+` after the parentheses (not a '*').